spring-boot/admin/src/main/resources/templates/admin/fragments/consumer_wizard/wizard.html

```diff
@@ -233,7 +233,7 @@ <h2>
     // Post call to subscribe the test service
     function subscribeButtonClicked() {
         $.post("/admin/services/consumer/subscribe", {
-            selectedServiceId: [[${testService.id}]]
+            selectedServiceId: [[${ testService != null ? testService.id: 0 }]]
```

Comment:
As there is not a service with the id equal 0 then I think this 0 here will create an exception

Reply:
I think this code doesnt make sense. This function should be available over one of the existing services to subscribe, so it should exist in advance. I will remove the condition of testService != null, also because if it is null, it will fail in the previous line "testService.id"

I dont reconigze this code

Reply:
fixed

Reply:
I agree that it should exist in advance, otherwise this whole functionality does not make any sense. That being said, the `0` seems very confusing to the reader of this code, as it does not have any real meaning. I would suggest to remove that "fallback `0`".

Reply:
It is Javascript code wrote by the parameter sent by the server, we should either fill with 0 or remove the complete blocked, becuase if not the Javascript code doesnt compile syntactically. It will result something like 
```

 $.post("/admin/services/consumer/subscribe", {
            selectedServiceId:
        }).done(function ChangeHTML() {
           
....
```

Reply:
OK, then I would suggest either of the following (just to make the situation clear):
```
   selectedServiceId: [[${ testService != null ? testService.id : "null" }]]
```
or if it needs a number:
```
   // testService will not be null if CoatRack is properly configured
   selectedServiceId: [[${ testService != null ? testService.id : -1 }]]
```